cyences_app_for_splunk/default/data/ui/views/cs_linux_user_groups_reports.xml

```diff
@@ -0,0 +1,127 @@
+<form version="1.1" theme="dark">
```

Comment:
- Can we combine this dashboard with the original Linux dashboard?

- Remove the previously created user details panel.

- Have the following panels: 

1. Groups (shows all the groups regardless of timerange along with group-members, created-time, and last updated time and hostname) 
2. Drilldown panel for Groups panel: show the full history of that group regardless of the time-range selected. The panel only visible when drilldown.
3. Group Changes: Show group changes only for selected time-range.
4. Users (shows all the users regardless of timerange along with group-name, hostname, created-time, and last updated time) 
5. Drilldown panel for Users panel: show the full history of that user regardless of the time-range selected. The panel is only visible when drilldown.
6. User Changes: Show user changes only for the selected time-range.

- Under all these panels add the below notes:
1. Time shown in this panel is the time of script execution and not accurate timing. Scripts execute every hour. Created-time might show the time from when the first time the script is executed on the host.
2. For the "Users" and "Groups" panels, add an extra note that, click on any row to show the full history of that group/user changes.

- Filters: Please keep the filters that you already have here. Looks good.

Reply:
Let's not use timerange for any panels. It will confuse user.